lib/src/scss-shared/_nx-radio-checkbox-helpers.scss

```diff
@@ -8,6 +8,6 @@
 // We are using box-shadow below instead of border
 // so we do not have to re-calculate the focus Pseudo-element in NxRadio and NxCheckbox
 // when the width changes in dark mode.
-@mixin inset-box-shadow($color, $border-width: var(--nx-border-width-radio-checkbox)) {
+@mixin inset-box-shadow($color, $border-width: var(--nx-border-width)) {
   box-shadow: inset 0 0 0 #{$border-width} #{$color};
 }
```

Comment:
Missing extra empty line at the end?

Reply:
> Missing extra empty line at the end?

Yup! e516493b8f6fea00118dca075357a1f270805785